src_kolko_krzyżyk/kolko.py

```diff
@@ -0,0 +1,76 @@
+class TicTacToe:
+    #class veribles
+    used_nums = ['','','','','','','','','']
+    board = [['0', '1', '2'],
+            ['3', '4', '5'],
+            ['6', '7', '8']]
+    way_to_win = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6],
+                  [1, 4, 7], [2, 5, 8], [0, 4, 8], [6, 4, 2]]
+
+    def __init__(self):
+        self.used_nums = self.used_nums
+        self.board = self.board
+        self.way_to_win = self.way_to_win
+
+    def printer(self):
+        'print board with positions to choose'
+        print('{}\n{}\n{}'.format(self.board[0], self.board[1], self.board[2]))
+
+    def instert_to_board(self, player, player_choice):
+        '''inserting player to board return update board '''
+        counter = 0
+        for arr in self.board:
+            for i, pos in enumerate(arr):
```

Comment:
You are not using `pos` in this code. I guess you wanted to express something like:
```
for index in range(len(arr)):
    if player_choice == counter:
        arr[index] = 'X' if player == 0 else '0'
    counter += 1
```